lib/cli/commands/connect.js

```diff
@@ -1,21 +1,21 @@
 const command = require('../command');
 
-exports.command = 'connect <host> [port]';
+exports.command = 'connect <p2p_host> [p2p_port]';
 
 exports.describe = 'connect to an xu node';
 
 exports.builder = {
-  host: {
-    type: 'number',
+  p2p_host: {
+    type: 'string',
   },
-  port: {
+  p2p_port: {
```

Comment:
We can add `description` properties here to clarify that these are the host and port of the remote peer we are connecting to.